etc/HIProdOfflineConfiguration.py

```diff
@@ -1483,7 +1483,7 @@
                                "HcalCalIsolatedBunchSelector", "HcalCalIterativePhiSym","HcalCalMinBias",
                                "TkAlJpsiMuMu", "TkAlUpsilonMuMu","TkAlZMuMu","TkAlMuonIsolated"],
                dqm_sequences=["@commonSiStripZeroBias", "@ecal", "@hcal", "@muon", "@jetmet", "@egamma"],
-               scenario=hiTestppScenario)
+               scenario=ppEra_Run3_2023)
```

Comment:
Should not this be `scenario=ppScenario` ?

See
https://github.com/dmwm/T0/blob/6d2579ffd3c86b3449c1bf3b0b44dca7f79e2689/etc/HIProdOfflineConfiguration.py#L105